### ISSUE TITLE
how to automatically generate the class body when createing then first java file without creating a java project

### ISSUE THREAD

**Issue by psyzhou** [redhat-developer/vscode-java#2657]
In vscode in windows, create the first java file in an empty folder directly, it will not automatically generate class body.

**Comment by fbricon** [comment]
that's expected because the Java extension starts only after the creation of the java file. However, you can type 'class' and access a snippet that'll fill the class body

**Comment by rgrunber** [comment]
Closing as a duplicate of https://github.com/redhat-developer/vscode-java/issues/2659. It's slightly older but that one has a bit more context (screenshot in original issue).
